docs/user_guide.rst

```diff
@@ -89,16 +84,37 @@ Checks
 ^^^^^^
 
 ``checks`` determines which alerts are checked and to whom notifications are
-sent.
+sent. Optionally, you can define an alert threshold (useful for twitchy checks).
+If you define an alert threshold, you should also define an all-clear threshold.
+If you use thresholds, an alert logger will be used for alert persistence. More
+information on the logger interface is in a section below::
 
+    // Simple implementation:
     'checks' => [
        'checkDB' => ['ops', 'devs'],
     ],
 
 In this case, ``checkDB`` is a global function that throws an
 :class:`AlertinatorException` when some alerting threshold is passed.  When
 that happens, the alert will be sent out to members of the ``ops`` and ``devs``
-:ref:`groups`.
+:ref:`groups`::
+
+    // Complex implementation, with alert thresholds:
+    'checks' => [
+        'checkDB' => [
+            'groups' => ['ops'],
+            'alertAfter'  => 5,
+            'clearAfter'  => 2,
+            'remindEvery' => 1,
+        ],
+    ],
+
+In this case, ``checkDB`` is still the check function, and when the alert
+exception is thrown, the alert will be sent to the ``ops`` group after the
+``afterAlert`` threshold is met. Also, once the check passes ``clearAfter``
+times in a row, an all-clear alert will be sent. Finally, the ``remindEvery``
+threshold limits reminders to only every X errors after the first alert is
```

Comment:
Suggested changes:

"\`\`afterAlert\`\`" -> "\`\`alertAfter\`\`"

"Finally, the \`\`remindEvery\`\` threshold limits reminders to only every X errors after the first alert is sent"
->
"Finally, reminder alerts are limited to only every \`\`remindEvery\`\` errors after the first alert is sent"

Reply:
Go for it! Otherwise I will next push.